packages/terra-site/src/examples/form-radio/RadioFieldIndex.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
lets use the terra-doc-template to create this file

Reply:
Wasn't sure if we could do that or not, since terra-site doesn't look to be on terra-dev-site. Nevertheless, I'll give it a whirl.

Reply:
terra-doc-template is it's own package that lives in terra-core. It's fine to use and has no dependency on terra-dev-site